woocommerce/payment-gateway/class-sv-wc-payment-gateway-payment-form.php

```diff
@@ -99,7 +99,22 @@ protected function add_hooks() {
 		add_action( "wc_{$gateway_id}_payment_form_end",   array( $this, 'render_fieldset_end' ), 5 );
 
 		// payment form JS
-		add_action( "wc_{$gateway_id}_payment_form_end",   array( $this, 'render_js' ), 5 );
+		add_action( 'wp_footer', [ $this, 'load_render_js' ] );
+	}
+
+
+	/**
+	 * Loads payment gateway render script if checkout page or pay page.
+	 *
+	 * @since 5.10.5
+	 */
+	public function load_render_js() {
+
+		// also make sure it's not order received page
+		if ( ! is_order_received_page() && ( is_checkout() || is_checkout_pay_page() ) ) {
```

Comment:
this is missing a `is_add_payment_method_page()` condition - the JS needs to load on My Account -> Add Payment Method as well.

Reply:
Yeap, make sense => 2341a97